src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -1,52 +1,138 @@
 import React, {FC} from 'react'
 import {useSelector} from 'react-redux'
-import {Link} from 'react-router-dom'
 
-import {Columns, Grid, InfluxColors, Page, Panel} from '@influxdata/clockface'
+import {
+  AlignItems,
+  ComponentSize,
+  FlexBox,
+  Icon,
+  IconFont,
+  JustifyContent,
+  Page,
+  ResourceCard,
+} from '@influxdata/clockface'
 
 import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
 import {getOrg} from 'src/organizations/selectors'
+import {
+  CLIIcon,
+  GoIcon,
+  JavascriptNodeJsIcon,
+  PythonIcon,
+  TelegrafIcon,
+} from 'src/homepageExperience/components/HomepageIcons'
+
+import './HomepageContainer.scss'
 
 export const HomepageContainer: FC = () => {
   const org = useSelector(getOrg)
   const pythonWizardLink = `/orgs/${org.id}/new-user-wizard/python`
+  const cliPageLink = `/orgs/${org.id}/load-data/file-upload/csv`
+  const telegrafPageLink = `/orgs/${org.id}/load-data/telegrafs`
+
+  const handleOpenPythonWizard = () => {
+    window.location.href = pythonWizardLink
+  }
+
+  const handleOpenCLI = () => {
+    window.location.href = cliPageLink
+  }
+
+  const handleOpenTelegraf = () => {
+    window.location.href = telegrafPageLink
+  }
```

Comment:
any reason why these are navigated to like this rather than being a react-router `Link` ?

Reply:
I just didn't realize I could wrap things inside of `Link`! Just fixed 👍